src/backend/shortcuts/ipc_handler.ts

```diff
@@ -63,8 +87,16 @@ ipcMain.handle(
     bkgDataUrl: string,
     bigPicDataUrl: string
   ) => {
-    const game = getGame(appName, runner)
-    const gameInfo = await game.getGameInfo()
+    let gameInfo: GameInfo
+
+    const isSideload = runner === 'sideload'
+    if (isSideload) {
+      gameInfo = getAppInfo(appName)
+    } else {
+      const game = getGame(appName, runner)
+      gameInfo = await game.getGameInfo()
+    }
```

Comment:
We have this pattern of checking if the runner is sideload or not and then get the gameInfo or the AppInfo in a lot of places (we also do that to get settings for example too). Maybe it's cleaner have a `getInfo(appName, runner)` and inside that method handle the ifSideload check? to avoid repetition

Reply:
That's true, will check if I can refactor this.

Reply:
Ok, added this handled and it is better now.
Not everywhere is possible and since we are using different approaches, Sideload does not depends on `Game` so it is hard to avoid repetition in some places, like the Shortcuts handlers, for instance.
Let's see if we can refactor the others runners in the future so we can cleanup this code.